src/pages/Home.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import {Link} from 'react-router-dom'
+import beerListimg from '../assets/beers.png'
+import randomBeer from '../assets/random-beer.png'
+import newBeer from '../assets/new-beer.png'
+
+
+function Home() {
+  return (
+    <div>
+        <Link to="/beer-list">
```

Comment:
João, you have the `Link to=""`'s wrapped around the `image`, the `h1` and the `p` tag. 
Not sure if it was supposed, but it's not a good practice.